docs/UserGuide.md

```diff
@@ -110,8 +114,9 @@ Deletes the specified item from the list of tracked inventory.
 Format: `deletei INDEX`
 
 * Deletes the item at the specified `INDEX`.
-* The index refers to the index number shown in the displayed inventory list.
-* The index **must be a positive integer** 1, 2, 3, …​
+
```

Comment:
I think this blank line causes some formatting inconsistencies in the document. Do remove it